doc/content/examples/example6a.md

```diff
@@ -0,0 +1,65 @@
+# Example 6a: Displacement Controlled Contact Problem with elastic soil
+
+## Model Description
+
+This example demonstrates the use of the contact block to simulate a frictional contact between a solid block and an elastic soil in MASTODON. A solid cubical block of dimensions 6" each side  is placed in contact with an cubical soil block of dimensions 48" each side. The remaining three sides of the soil are fixed and the solid block is displaced by a prescribed distance in the direction parallel to the contact surface. The normal forces and the frictional forces obtained from MASTODON is compared with the analytical results.
```

Comment:
This example demonstrates frictional contact between a solid block and elastic soil in MASTODON. A 6 inch cubic block is centered on top of a 4 foot cubic block of soil. The top surface of the soil is free and the remaining surfaces are fixed in all directions. A uniform normal pressure of 5 psi is applied to the top of the smaller solid block and it is given a prescribed displacement in a direction parallel to the contact surface. The resulting normal and frictional forces at the interface of the two materials are then obtained and compared with analytical results.